packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/logs-encryption-keys.controller.js

```diff
@@ -0,0 +1,6 @@
+export default class LogsEncryptionKeysCtrl {
+  /* @ngInject */
+  constructor($stateParams) {
+    this.$stateParams = $stateParams;
```

Comment:
**nitpick:** Avoid using `$stateParams` in controller files as this usage makes the migration to other frameworks difficult. You can have functions defined in `resolve` of routing files and inject it using component input.

Reply:
Fixed